src/main/java/Javatro/HoldingHand.java

```
@@ -10,32 +10,53 @@
 
 public class HoldingHand {
     protected List<Card> Hand;
-    private final int HOLDINGLIMIT = 8; // The maximum number of cards a hand can hold
+    private final int HOLDING_LIMIT = 8; // The maximum number of cards a hand can hold
 
-    public HoldingHand(Deck deck) {
+    /**
+     *
+     * Instantiate an empty List of Cards.
+     */
+    public HoldingHand() {
         Hand = new ArrayList<Card>();
-        for (int i = 0; i < HOLDINGLIMIT; i++) {
-            Hand.add(deck.draw());
+    }
+
+    /**
+     * Adds a Card to the Hand.
+     * <p>
+     * This function should not be called if Hand size >= HOLDING_LIMIT
+     */
+    public void add(Card cardToAdd) {
+        if (Hand.size() < HOLDING_LIMIT) {
+            Hand.add(cardToAdd);
+        } else {
+            System.out.println("Exceeded Limit. Card not added.");
```

Maybe throw an exception here for this.